File: Manuel/2025-07-31/숫자의 개수/count_numbers.py
import sys

def solve():
    A = int(sys.stdin.readline())
    B = int(sys.stdin.readline())
    C = int(sys.stdin.readline())
    product = A * B * C
    str_product = str(product)

    counts = [0] * 10
    for char_digit in str_product:
        int_digit = int(char_digit)
        counts[int_digit] += 1
    
    for i in range(10):
        print(counts[i])

solve()

File: Manuel/2025-07-31/숫자의 개수/count_numbers.cpp
#include <iostream>
#include <vector>
#include <string>

using namespace std;

int main()
{
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);

    long long A, B, C;
    cin >> A >> B >> C;
    long long product = A * B * C;

    string str_product = to_string(product);
    vector<int> counts(10, 0);

    for(char char_digit : str_product)
    {
        int digit = char_digit - '0';
        counts[digit]++;
    }

    for(int i = 0; i < 10; i++)
    {
        cout << counts[i] << "\n";
    }

    return 0;
}